setup.cfg

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python                                                           
+# -*- coding: utf-8 -*-                                                         
+#                                                                               
+# Copyright (c) 2019 VMware, Inc. All Rights Reserved.                          
+# SPDX-License-Identifier: BSD-2-Clause
+
+[metadata]
+name = tern
+author = VMWare Inc
+author-email = FIXME@FIXWHAT.THEEMAIL
```

Comment:
We'll need to change this to a legit email

Reply:
Done! Changed to Nisha's email.